packages/database/test/exp/integration.test.ts

```diff
@@ -138,6 +141,43 @@ describe('Database@exp Tests', () => {
     unsubscribe();
   });
 
+  it('can connect to emulator', async () => {
```

Comment:
Should we just wrap all of these in `if (USE_EMULATOR)`?

Reply:
Done!